content/java/concepts/math-methods/terms/acos/acos.md

```diff
@@ -11,11 +11,11 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `Math.acos()` method returns the inverse cosine of the argument in radians.
+The **`Math.acos()`** method returns the inverse cosine of the argument in radians.
 
 ## Syntax
 
-```java
+```pseudo
 Math.acos(n)
 ```
 
```

Comment:
Can we add a brief sentence here describing what `n` is (including what data type(s))?

Reply:
I'm also wondering if we should combine the two "Example [x]" section into one? Maybe something like this:

```java
class Main {
  public static void main(String[] args) {
    double arg1 = 0.0;
    double arg2 = 1.0;

    System.out.println(Math.acos(arg1));
    System.out.println(Math.acos(arg2));
  }
}
```

And then we could follow up with some text that precedes a \```shell output snippet:

"This will yield the following output:

```shell
1.5707963267948966
0.0
```
"